plugins/flutter_aepassurance/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.0.0
+
+* Update to use BOM (Bill of Materials) for sdk dependecies in Android
```

Comment:
Update to use BOM ([Bill of Materials](https://central.sonatype.com/artifact/com.adobe.marketing.mobile/sdk-bom)) for Android SDK dependencies.